static/js/overlays.js

```diff
@@ -36,6 +36,20 @@ exports.drafts_open = function () {
     return open_overlay_name === 'drafts';
 };
 
+// This just adds the inline style to '.overlay.show'
+// and overrides the "pointer-events: all" style in
+// app_components.scss.
```

Comment:
Let's explicitly say that the `.overlay.show` refers to any overlay that's possibly underneath the current modal.  I think that's what we're affecting here, correct?

Reply:
Thanks, the rewrite works for me!